modules/core/arrow-generic/src/test/kotlin/arrow/generic/CoproductTest.kt

```diff
@@ -82,5 +82,37 @@ class CoproductTest : UnitSpec() {
                     { "Third" }
             ) shouldBe "Third"
         }
+
+        "cop replacement functions are equivalent" {
+            val firstValue = "String"
+            val secondValue = 100L
+            val thirdValue = none<String>()
+
+            firstValue.cop<String, Long, Option<String>>() shouldBe firstValue.first<String, Long, Option<String>>()
```

Comment:
didn't we remove cop ext funs?

Reply:
No I think the intent is to deprecate them and remove them in a later version? https://github.com/arrow-kt/arrow/issues/1284#issuecomment-460337722

I'm not sure what the next Arrow version will be I guess, I may have misunderstood

Reply:
that is correct @abergfeld they will be deprecated in 0.9.0 coming out now and will be removed sometime before 1.0 final.